SPECS/kernel/kernel.spec

```diff
@@ -15,6 +15,7 @@ Source0:        kernel-%{version}.tar.gz
 Source1:        config
 Source2:        config_aarch64
 Source3:        sha512hmac-openssl.sh
+Source4:        cbl-mariner-ca-20210127.pem
```

Comment:
I'm assuming these have to be always the same for this and the Hyper-V version of the kernel. I don't have a quick idea for a check/automation, which will make sure we're always using the same files, but I'd suggest at least adding a comment explaining that here and inside the Hyper-V's spec.

Alternatively, we could use a symbolic link and use only one "actual" file, but I'm not sure I'd like to make that precedence.

Reply:
It is not necessarily required that these certificates always be the same.